sphinx_automodapi/utils.py

```diff
@@ -81,6 +81,9 @@ def find_mod_objs(modname, onlylocals=False):
     else:
         pkgitems = [(k, getattr(mod, k)) for k in dir(mod) if k[0] != '_']
 
+    if sort:
```

Comment:
Given that `dir` is guaranteed to sort, this could be moved inside the `if` statement. (This is true even if `__dir__` is defined and returns something unsorted.)